### PR TITLE
quick bug fix on REGISTRY STRING

### PR DESCRIPTION
**Pull request by abanna**
## 📚 Purpose
Fixed an issue with REGISTRY env var when building images.  The script will now add the trailing '/' or leave it if the user provides it in the string. In addition, the script will now exit if the user attempts to push images to Docker hub.

## 👌 Resolves:
- [x] 🐛 CLOUD-446

## 📦 Impacts:
pnpm image
pnpm images
pnpm push

### CODE REVIEW

File: bin/build-helper [hunk 1]
```diff
@@ -17,9 +17,21 @@ createImagesFileIfNotExists() {
   fi
 }
 
+setRegistry() {
```
Comment (janedotx) [inline bin/build-helper:20]: Maybe rename to `formatRegistry`, since the registry value's actually set in the env?

Comment (abanna) [reply bin/build-helper]: This has been updated.

File: bin/build-helper [hunk 2]
```diff
@@ -32,10 +44,18 @@ buildImage() {
 
 pushImages() {
   createImagesFileIfNotExists
-  cat /dev/null > $BASE_DIR/dist/images.txt
+
+  if [[ $REGISTRY == "" ]]; then
+    echo "REGISTRY env var is not set. Please add REGISTRY to env or as a var before the command."
+    exit 1
+  else
+    formatRegistry
```
Comment (invertigo) [inline bin/build-helper:52]: I don't think this needs to be here - `formateRegistry` gets called in the `buildImage` function before writing to `images.txt`

**Review by invertigo — CHANGES_REQUESTED** [review]

File: bin/build-helper (hunk 2 above)
Comment (abanna) [reply bin/build-helper:52]: I think you are correct sir! I had originally planned to validate the content of the images.txt file.

Comment (abanna) [reply bin/build-helper]: fixed.

**Review by avatarneil — APPROVED** [review]

**Review by nessur — APPROVED** [review]

**Pull request closed without merging**